notebooks/time-to-merge-prediction/workshop/00_read_env.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>Can we also add an explanation for why should someone put <code>REMOTE = 1</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>+1</p>